lib/pengine/rules.c

```diff
@@ -553,8 +553,17 @@ pe_unpack_nvpairs(xmlNode *top, xmlNode *xml_obj, const char *set_name,
                     always_first, overwrite, next_change);
 }
 
+/*!
+ * \brief Expand any regular expression submatches (%0-%9) in a string
```

Comment:
No issue with this per se, but the Pacemaker Explained doc could use an improved explanation (ideally with an example or two) of how regexes and submatches work with regard to rules. Examples would probably help elucidate the purpose of these utilities too.

Reply:
Definitely overdue, added commit on top. Generated docs viewable at https://people.redhat.com/~kgaillot/pacemaker/doc/